src/assets/js/react/components/FontManager/FontVariantLabel.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const FontVariantLabel = ({ label, font }) => (
+  <div htmlFor={'gfpdf-font-variant-' + label}>
+    {label === 'regular' && font === 'false' && <span>Regular <span className='required'> (required)</span></span>}
+    {label === 'regular' && font === 'true' && 'Regular'}
+    {label === 'italics' && 'Italic'}
+    {label === 'bold' && 'Bold'}
+    {label === 'bolditalics' && 'Bold Italic'}
```

Comment:
These strings need to be moved to `Helper_Data` and accessed via `GFPDF`

Reply:
I feel like this function could be done in a clearer way too. Only the Regular font has special syntax, everything is just a id => label mapping.